oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/user/DefaultOAuth2User.java

```diff
@@ -127,10 +128,34 @@ public String toString() {
 		sb.append("Name: [");
 		sb.append(this.getName());
 		sb.append("], Granted Authorities: [");
-		sb.append(this.getAuthorities().stream().map(GrantedAuthority::getAuthority).collect(Collectors.joining(", ")));
```

Comment:
I think this might be simpler as:

```java
sb.append("], GrantedAuthorities: ");
sb.append(getAuthorities())
sb.append(", User Attributes: ");
sb.append(getAttributes());
return sb.toString();
```

Reply:
I can't change function names from collectAuthorities to getAuthorities, cause there are exists overriding functions with this names, but I change the params.